supabase/datadog_checks/supabase/metrics.py

```diff
@@ -0,0 +1,248 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base.checks.openmetrics.v2.metrics import DEFAULT_GO_METRICS
```

Comment:
The only thing I'd say we should do differently here is to change this metric:
https://github.com/DataDog/integrations-core/blob/642b2f9f30a05c09f32025a9176d0e1a13cf8f61/datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/metrics.py#L31

We should make it an elapsed time metric. I can do that. But then we'd have to make a release and bump the min base package in this check. The easier way maybe to overwrite the metric in your metric list.

Reply:
I made the overwrite and the PR to fix it in the base class. I'm also adding some missing metrics to both.

Reply:
This one should also be changed:
https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/metrics.py#L14